cmake/Modules/KatanaBuildCommon.cmake

```diff
@@ -23,6 +23,7 @@ set(KATANA_ENABLE_PAPI OFF CACHE BOOL "Use PAPI counters for profiling")
 set(KATANA_ENABLE_VTUNE OFF CACHE BOOL "Use VTune for profiling")
 set(KATANA_STRICT_CONFIG OFF CACHE BOOL "Instead of falling back gracefully, fail")
 set(KATANA_GRAPH_LOCATION "" CACHE PATH "Location of inputs for tests if downloaded/stored separately.")
+set(KATANA_ENABLE_COVERAGE OFF CACHE BOOL "Add instrumentation (used for code coverage collection) to binaries.")
```

Comment:
Separate PR: It would be nice to have this flag automatically enable using `coverage` for python when called from ctest (since ctest also calls pytest). That way a single ctest run could get coverage information for both languages.

Reply:
Was thinking about this.  Currently, the only thing one would have to do is to export an env variable for python before the build, and then you get both: coverage for Python and coverage for C++.  I feel being able to turn on/off each individually is good from users perspective, but we could also provide a single option in the future that simply means both (which is what you were suggesting).

Reply:
Maybe this option should be a list of language names in the future.

Reply:
That would be indeed more uniform than env variable + an option.